### DISCUSSION TITLE
如果想忽略某些警告而继续编译脚本应该怎样做？

### DISCUSSION THREAD

**Discussion by sillsun** [dotnetcore/Natasha#209] [Q&A]
你好，我使用Natasha对脚本进行编译，有些警告如果使用vs是可以直接编译，例如 : NatashaException:““xxx.property”隐藏继承的成员“x.property”。如果是有意隐藏，请使用关键字 new。”。

请问我应该要怎样配置AssemblyCSharpBuilder才能忽略这种警告然后进行编译？

**Comment by NMSAzulX** [comment]
@sillsun 试试一下代码, 另外你的脚本是啥样的, 方便复现一下这个错误吗?
```c#
AssemblyCSharpBuilder cSharpBuilder = new();
 cSharpBuilder.ConfigCompilerOption(item => item.AddSupperess("CS8019").UseSuppressReportor(false));
```

**Reply by NMSAzulX** [reply]
我这边是编译通过的, 5.2.2.1 版本.
